app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,45 @@
+class GuestsController < ApplicationController
+  before_action :require_login, only: [:new, :create]
+
+  def show
+
+  end
+
+  def new
+    @guest = Guest.new
+    respond_to do |format|
+      if request.xhr?
+        format.html { render :new, layout: false}
+      else
+        format.html { render :new }
+      end
+    end
+  end
+
+  def create
+    new_guest = Guest.new(get_params)
+    new_guest.user_id  = current_user.id
+    if new_guest.save
+      flash[:success] = "#{new_guest.name} was saved"
+      redirect_to user_path(current_user)
+    else
+      flash[:error] = "This guest could not be saved"
+      redirect_to :back #ask about changing this
+    end
+  end
+
+  def update
+
+  end
+
+  def delete
+
```

Comment:
If this is not doing anything, delete it. Exception: if there is a corresponding view it's kind of OK to leave the empty action as a visual cue that the view/route exists. But it seems unlikely that's the case with a delete.